src/how-to-build-and-run.md

```diff
@@ -385,7 +385,7 @@ should see a version number ending in `-dev`, indicating a build from
 your local environment:
 
 ```bash
-> rustc +stage1 -vV
+$ rustc +stage1 -vV
```

Comment:
Hmm I see why you added $ there but I'm not sure what to think about this.
cc @mark-i-m